docker/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM alpine:3.6
+
+# ply is installed to allow cxxtestgen to be used with the FOG parser for
+# test discovery.
+# See http://cxxtest.com/guide.html#_test_discovery_options
+RUN apk update \
+  && apk add python3=3.6.1-r2 \
+             g++=6.3.0-r4 \
+             make=4.2.1-r0 \
+  && ln -s /usr/bin/python3 /usr/bin/python \
+  && pip3 install ply==3.10
+
+COPY cxxtest /cxxtest
+ENV PATH="/cxxtest/bin:${PATH}" CXXTEST="/cxxtest"
+
+RUN /bin/sh
```

Comment:
if you have bash, I'd rather run it than sh.

Reply:
+1

Reply:
I don't think Alpine has `bash`. Does it?

Reply:
Probably not, good point. It can be installed though. Depends what it will be needed for (`sh` is rather annoying to use ^^)

Reply:
The Dockerfile now installs bash and I've removed the RUN (which did not make a lot of sense) in favor of an ENTRYPOINT.